src/routes/safe/store/selectors/index.js

```diff
@@ -98,6 +98,25 @@ export const safeTransactionsSelector: TxSelectorType = createSelector(
   },
 )
 
+export const creationTxSelector: TxSelectorType = createSelector(
+  safeTransactionsSelector,
+  (transactions: List<Transaction>): Transaction => {
+    if (!transactions.size > 0) {
+      return null
+    }
+
+    const creationTx = transactions.filter((tx) => {
```

Comment:
1. you can use `.find`
2. I think we can have a non-creation tx with nonce 0

Reply:
That selector wasn't needed anymore, I just removed it